src/Sarif.Viewer.VisualStudio/ProjectNameLookup.cs

```diff
@@ -0,0 +1,33 @@
+﻿using System.Collections.Generic;
+
+namespace Microsoft.Sarif.Viewer
+{
+    public sealed class ProjectNameLookup
```

Comment:
>ProjectNameLookup [](start = 24, length = 17)

I'm not crazy about this name, since class names are usually nouns, and this doesn't look like a noun.

What do you think of `ProjectNameCache`?